jwasmer/trivia
# Feature/app

## What does this PR do?

Functionality for App.js

- [x] Component skeleton 
- [x] Fetch()
- [x] State that holds allData object
- [x] Function for keeping score 

Data/State for App.js

  - [x]  correct guesses by continent
  - [x]   incorrect guesses by continent 

Continents object for App.js

- [x] Continents
  - [x] Continents name
  - [x] Countries
  - [x] Languages
  - [x] Capitals
  - [x] Emoji (flag)
  - [x] Country name

## How to test
Pull branch then:
1. Check that the `Fetch()` is working with a console log
2. Look for typescript errors or data missing typescript
3. Ensure that all items listed above are handled

## What (if any) features are you implementing
See above

## What (if anything) did you refactor?
None

## Were there any issues that arose?
None

## Is there anything that you need from your teammate?
Nope 

## Any other comments, questions, or concerns?
I am making a separate ticket for` React Router `
